README.md

```diff
@@ -83,7 +83,7 @@ You can set up your GitHub Pages to deploy your repository.
 ```
 npm run release --branch [branchname] # can contain /
 ```
-2. Now, your repository can be found at http://<username>.github.io/<repository-name>/<branch-name>
+2. Now, your repository can be found at https://YOUR_GITHUB_USERNAME.github.io/binary-bot/YOUR_BRANCHNAME/
```

Comment:
hey Amina, i think it is a bit different
when i run the script my deploy is at https://nikolai-deriv.binary.sx/BRANCH_NAME/bot.html
e.g.: https://nikolai-deriv.binary.sx/add-languages/bot.html
```suggestion
2. Now, your repository can be found at https://YOUR_GITHUB_USERNAME.binary.sx/binary-bot/YOUR_BRANCHNAME/bot.html
```